### PR TITLE
Documentation - Add Antennas and Gain Pattern Videos

### PR DESCRIPTION
**Pull request by jonpas**
**When merged this pull request will:**
- Move Class Names page from User to Frameworks section in sidebar (URL remains the same)
- Add Antenna class names
- Add Antennas list to Basic Concepts page
- Add Antenna Gain Pattern videos by @Drofseh 
- Rework Basic Concepts page structure/order

image

### CODE REVIEW

**Comment by jonpas** [comment]
Approved by @Drofseh.

**Review by Drofseh — APPROVED** [review]

**Comment by jonpas** [comment]
Updated with a little description about gain patterns and their use (inspired by chat with @Drofseh).

image

File: docs/wiki/class-names.md [hunk 1]
```diff
@@ -20,8 +20,24 @@ title: Class Names
...
+- `ACRE_2HALFINCH_UHF_TNC` - 2.5 INCH UHF PRC343
+- `ACRE_SEM52_SHORT_BNC` - 0.4M SEM52/SEM70
+- `ACRE_SEM52_LONG_BNC` - 0.9M SEM52
+- `ACRE_SEM70_LONG_BNC` - 1.03M SEM70
+- `ACRE_AT271_38` - AT-271 PRC-77
+- `ACRE_123CM_VHF_TNC` - 1.23M VHF TNC
+- `ACRE_270CM_VEH_BNC` - 2.7M MB VEH BNC
+- `ACRE_FA80_BNC` - FA80 VEH BNC
+- `ACRE_AS1729_38` - AS-1729/VRC
+- `ACRE_243CM_VHF_TNC` - 2.43M Ground Spike
```
Comment (TheMagnetar) [inline docs/wiki/class-names.md:42]: This gets confusing with the tittle separation. It is an antenna but its description is ground spike

File: docs/wiki/user/basic-concepts.md [hunk 2]
```diff
@@ -30,3 +33,27 @@ Unlike light these radio waves can still penetrate things like walls to some ext
...
+| [2.5 INCH UHF PRC343](https://www.youtube.com/watch?v=UZQZ23aorg4) | TNC | **AN/PRC-343** |
+| [0.4M SEM52/SEM70](https://www.youtube.com/watch?v=9K7nYbC4tD8) | BNC | SEM 52 SL |
+| [0.9M SEM52](https://www.youtube.com/watch?v=N72diKJHo5k) | BNC | **SEM 52 SL** |
+| [1.03M SEM70](https://www.youtube.com/watch?v=t3YMP7XuIwM) | BNC | **SEM 70** |
+| [AT-271 PRC-77](https://www.youtube.com/watch?v=v0EGHoT7XGA) | 3/8" | **AN/PRC-77** |
+| [1.23M VHF TNC](https://www.youtube.com/watch?v=a-ydqMUjAMY) | TNC | **AN/PRC-117F** |
+| [2.7M MB VEH BNC](https://www.youtube.com/watch?v=zALwqsWz7Wk) | BNC | **AN/VRC-103**, **AN/VRC-110**, **AN/VRC-111** |
+| [FA80 VEH BNC](https://www.youtube.com/watch?v=rubFrJwYhgg) | BNC | **SEM90** |
+| [AS-1729/VRC](https://www.youtube.com/watch?v=3G_87lPpEMM) | 3/8" | **AN/VRC-64** |
+| [2.43M Ground Spike](https://www.youtube.com/watch?v=GDZLo0TkZyI) | TNC | AN/PRC-148, AN/PRC-152, AN/PRC-117F, AN/PRC-77, SEM 70 |
```
Comment (TheMagnetar) [inline docs/wiki/user/basic-concepts.md:56]: Same as above

**Review by TheMagnetar — CHANGES_REQUESTED** [review]

File: docs/wiki/class-names.md (hunk 1 above)
Comment (jonpas) [reply docs/wiki/class-names.md:42]: These are all using `shortName` from the antenna config:
https://github.com/IDI-Systems/acre2/blob/29e20ea74051eb1274735c8d28a04fa55eaf2ba7/addons/sys_antenna/CfgAcreComponents.hpp#L121
The full name isn't any better. What do you suggest?

File: docs/wiki/user/basic-concepts.md (hunk 2 above)
Comment (jonpas) [reply docs/wiki/user/basic-concepts.md:56]: https://github.com/IDI-Systems/acre2/blob/29e20ea74051eb1274735c8d28a04fa55eaf2ba7/addons/sys_antenna/CfgAcreComponents.hpp#L121

**Review by TheMagnetar — APPROVED** [review]

**Pull request merged**